src/views/checkups/network/components/form/CheckupsNetworkFormNads.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { Dispatch, FC, SetStateAction, SyntheticEvent, useState } from 'react';
+
+import useNADsData from '@kubevirt-utils/components/NetworkInterfaceModal/components/hooks/useNADsData';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { isEmpty } from '@kubevirt-utils/utils/utils';
+import { useActiveNamespace } from '@openshift-console/dynamic-plugin-sdk';
+import { Dropdown, DropdownItem, DropdownToggle, FormGroup } from '@patternfly/react-core';
+
+type CheckupsNetworkFormNadsProps = {
+  selectedNad: string;
+  setSelectedNad: Dispatch<SetStateAction<string>>;
+};
+const CheckupsNetworkFormNads: FC<CheckupsNetworkFormNadsProps> = ({
```

Comment:
NAD is an abbreviation. I thought our convention was to use all capitals for abbreviations?

Reply:
i saw in a few places in code where we used Nads, but sure, lets cut this mistake